feed/models/post.py

```diff
@@ -17,3 +18,6 @@ class Post(models.Model):
 
     def __str__(self):
         return self.title
+
+    def get_absolute_url(self):
+        return reverse('feed')
```

Comment:
@paOmer could you please explain why is this needed, do we need to add this in every app?

Reply:
In this case the create new post form has been created with Django's CreateView class.
After successfully submitting such a form, the CreateView class is looking to redirect the user to another page in the website, its default way is to look for such a function in the model file.

This do not need to be added to every app or model in the project, only the ones that are relevant.

Reply:
I understand, thank you for the explanation.